### PR TITLE
Added Jade language

### PR DESCRIPTION
**Pull request by arfatbk**
Added hello world template for Jade

### CODE REVIEW

File: Jade/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Jade 
+### Jade is a node templating engine mostly used for generating plain HTML files
```
Comment (sbshah97) [inline Jade/README.md:2]: Maybe you can remove the `###` here

File: Jade/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+##### 1. Download and install jade from [Jade-lang.com](jade-lang.com) via npm as below
+   `$ npm install jade --global`
+   
+##### 2.Write your `Jade code`using any text editor, like
```
Comment (sbshah97) [inline Jade/README.md]: Maybe you can remove the `#####` here and replace it with `*`

File: Jade/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  * [Brackets](brackets.io)
+  * [SublimeText](sublimetext.com)
+  
+##### 3. Save file with extension `.jade`
```
Comment (sbshah97) [inline Jade/README.md]: Maybe you can remove the `#####` here and replace it with `*`

File: Jade/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Jade 
+### Jade is a node templating engine mostly used for generating plain HTML files
+
+##### 1. Download and install jade from [Jade-lang.com](jade-lang.com) via npm as below
```
Comment (sbshah97) [inline Jade/README.md]: Maybe you can remove the `#####` here and replace it with `*`

File: Jade/README.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+   
+##### 2.Write your `Jade code`using any text editor, like
+  * [Brackets](brackets.io)
+  * [SublimeText](sublimetext.com)
+  
+##### 3. Save file with extension `.jade`
+For example,
+`index.jade`
+
+##### 4. Compile code to generate plain HTML using this command
```
Comment (sbshah97) [inline Jade/README.md]: Maybe you can remove the `#####` here and replace it with `*`

File: Jade/README.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+`$ jade index.jade`
+
+##### Now you have `index.html` file, if you done everything correctly, You can open this `index.html` file in browser by double clicking it.
```
Comment (sbshah97) [inline Jade/README.md]: Maybe you can remove the `#####` here and replace it with `*`

**Review by sbshah97 — CHANGES_REQUESTED** [review]
Make the required changes in the README and I'll merge your PR!

**Review by arfatbk — COMMENTED** [review]
Updated README.md. This might do

File: Jade/README.md (hunk 1 above)
Comment (sbshah97) [reply Jade/README.md:2]: I think you missed this!

**Review by sbshah97 — COMMENTED** [review]
One last change :p

**Comment by arfatbk** [comment]
Do you want me to remove headings and make them bullets???

**Review by sbshah97 — APPROVED** [review]

**Comment by sbshah97** [comment]
I think it's fine! Good work! Thank you for contributing!

**Pull request merged**